_replication/replication-methods/log-based-incremental-replication.md

```diff
@@ -0,0 +1,260 @@
+
+      {{ site.data.tooltips.log-based-incremental-rep }} A binary log file is a record of events that occur within a database.
+
+      In this guide, we'll cover:
+
+      1. [How it works (with examples)](#how-log-based-incremental-replication-works),
+      2. [When it should be used](#when-log-based-incremental-replication),
+      3. [Limitations of this Replication Method](#limitations), and
+      4. [How to enable it for your database integration](#enabling-log-based-replication)
+
+  - title: "How {{ page.title }} works"
+    anchor: "how-log-based-incremental-replication-works"
+    content: |
+      There are two types of binary log replication: Statement and row-based. Stitch uses a row-based approach, which means that when rows are modified via [a supported event type](#limitation-2--database-event-types), the entire row is written to the binary log file as a log message.
+
+      During a replication job, Stitch iterates over each message in sequence, or in the same order that the messages were written to the binary log. Data is extracted for any messages that correspond to tables that have been tracked for replication. This means that all tables using {{ page.table }} are extracted from the source database simultaneously as one seamless process.
+
+      To ensure only new and updated data is selected for replication, Stitch will use a binary log's Log Sequence Number (LSN) to 'bookmark' its place in a log file. This value is used to resume replicating data where a previous replication job ended.
+
+    subsections:
+      - title: "Initial replication jobs"
+        anchor: "initial-replication-jobs"
+        content: |
+          During the initial replication job for a table using {{ page.title }}, Stitch will use a `SELECT`-based approach to replicate the table in full. If this were a SQL query, it would look like this:
+
+          ```sql
+          SELECT *
+            FROM schema.table
+          ```
+
+      - title: "Ongoing replication jobs"
+        anchor: "ongoing-replication-jobs"
+        content: |
+          After the historical replication of a table is complete, Stitch will read updates for the table from the database's binary logs. During ongoing replication jobs using {{ page.title }}, a few things will happen:
+
+          1. Stitch reads log messages in the binary file and persists data for tables set to replicate.
+          2. At the end of the replication job, Stitch bookmarks its place in the log file by storing its current LSN.
+          3. During the **next** replication job, Stitch will resume reading data with a **greater** LSN than the LSN from the previous job.
+          4. At the end of the replication job, Stitch bookmarks its place in the log file again.
+          5. Repeat.
+
+          While {{ page.title }} isn't a SQL-based approach, the following SQL query demonstrates how ongoing replication jobs using {{ page.title }} would look as SQL:
```

Comment:
I honestly think that this will confuse people

Reply:
I wondered about this as well. We can scrap it.